lib/core.ps1

```diff
@@ -21,7 +21,9 @@ $cachedir = $env:SCOOP_CACHE, "$scoopdir\cache" | Select-Object -first 1
 # Note: Github disabled TLS 1.0 support on 2018-02-23. Need to enable TLS 1.2
 # for all communication with api.github.com
 function enable-encryptionscheme($scheme) {
-  [Net.ServicePointManager]::SecurityProtocol = [Net.ServicePointManager]::SecurityProtocol -bor $scheme
+    # Net.SecurityProtocolType is a [Flags] enum, binary-OR sets
+    # the specified scheme in addition to whatever scheme is already active
+    [Net.ServicePointManager]::SecurityProtocol = [Net.ServicePointManager]::SecurityProtocol -bor ([Net.SecurityProtocolType]$scheme)
```

Comment:
`[Net.SecurityProtocolType]$scheme` will become `[Net.SecurityProtocolType]"Tls12"`, does it work? Why not `[Net.SecurityProtocolType]::Tls12`

Reply:
Because it is a function that accepts `$scheme`.

The function can be called with other parameters in the future.